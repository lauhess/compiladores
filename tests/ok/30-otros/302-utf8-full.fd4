let (x:Nat) = print "Prueba: á ñ 😜 海賊王 " 123