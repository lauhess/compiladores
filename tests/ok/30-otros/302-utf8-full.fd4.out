Prueba: á ñ 😜 海賊王 123
